forte/data/readers/sem_eval_task8_reader.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The reader that reads prodigy text data with annotations into Datapacks.
+"""
+
+import os
+from typing import Any, Iterator
+from forte.data.data_pack import DataPack
+from forte.data.readers.base_reader import PackReader
+from forte.data.data_utils_io import dataset_path_iterator
+from ft.onto.base_ontology import Sentence, RelationLink, EntityMention
+
+__all__ = [
+    "SemEvalTask8Reader"
+]
+
+
+class SemEvalTask8Reader(PackReader):
+    r""":class:`SemEvalTask8Reader` is designed to read in
```

Comment:
Please include the citation of the paper, that will be more permanent than web link.